### PR TITLE
release notes cluster checks

### PR DESCRIPTION
**Pull request by nslater**
this adds the cluster checks docs for 1.1

### CODE REVIEW

**Comment by nslater** [comment]
#5105 should be merged before this one. and then this pr should be rebased against the `release-notes-restructure` story branch

**Comment by nslater** [comment]
@matriv when we spoke in person you mentioned that there was somewhere in the docs main that could be linked through to the new release notes for 1.1. unfortunately I didn't write it down and I've been unable to find the section you pointed out to me

**Comment by matriv** [comment]
@nslater the cluster checks are here:
https://github.com/crate/crate/blob/master/blackbox/docs/sql/system.txt#L1168

File: blackbox/docs/release_notes/1/1/index.txt [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+For each table, run a command like so::
+
+  cr> OPTIMIZE TABLE table_ident WITH (upgrade_segments=true);
+
+Here, replace ``table_ident`` with the name of the table you are upgrading.
+
+Once you have done this for all tables, you can move on.
+
+Verify Migration
```
Comment (matriv) [inline blackbox/docs/release_notes/1/1/index.txt]: Verify Upgrade Process

File: blackbox/docs/release_notes/1/1/index.txt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+````````````````
+
+Open your CrateDB `admin console`_ and verify that your cluster checks are
+green. This indicates that all tables have been migrated.
```
Comment (matriv) [inline blackbox/docs/release_notes/1/1/index.txt]: all tables have been successfully upgraded or recreated are compatible with future versions of CrateDB.

File: blackbox/docs/release_notes/1/1/index.txt [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+If your cluster checks are not green, you may have missed some tables. Please
+go back to the the start.
+
+Create New Snapshots
+````````````````````
+
+Snapshots of your tables created prior to them being migrated will not work
```
Comment (matriv) [inline blackbox/docs/release_notes/1/1/index.txt]: upgraded or recreated

**Pull request closed without merging**